src/Libraries/DSOffice/Excel.cs

```diff
@@ -82,10 +84,6 @@ public static Application RegisterAndGetApp()
             if (excel == null)
             {
                 excel = new Microsoft.Office.Interop.Excel.Application();
-                if (excel == null)
-                {
-                    throw new Exception("Excel could not be opened.");
-                }
```

Comment:
if we are `new`ing an object, it's not really necessary to do a `null` check for it. Resharper therefore suggests that the code block I removed here is extraneous.